packages/docs/src/components/content/MaturityChecklist/Status.tsx

```diff
@@ -13,7 +13,7 @@ const Status = ({ status }: StatusProps) => {
   if (status === true) {
     return (
       <>
-        <SvgIcon ariaHidden title="" viewBox="0 0 24 24">
+        <SvgIcon ariaHidden title="" viewBox="0 0 24 24" className="ds-u-color--success">
```

Comment:
@hi-its-hailey (commenting here because github won't let me comment the actual line i have questions on) it looks like the "Complete," "N/A," "Incomplete" labels we use for the status icons have intentionally been set to `ds-text-body--sm` (line 22 of this file) but the mocks show these labels having the same size as the body text. 

should we keep the original smaller font size or should i remove the util class? i can show mocks next week if needed.

Reply:
@zarahzachz Let's keep the original smaller font size!